package.json

```diff
@@ -57,6 +57,9 @@
   "standard": {
     "env": [
       "mocha"
+    ],
+    "ignore": [
+      "docs/swagger-ui/"
```

Comment:
Ignore the swagger-ui folder since it is not related to the linting of the source code.